doc/source/examples/dem/small-scale-rotating-drum-post-processing/small-scale-rotating-drum-post-processing.rst

```diff
@@ -0,0 +1,216 @@
+
+==========================================
+Small scale rotating drum post-processing
+==========================================
+
+This is an example of how to post-process results obtained in the `Small scale rotating drum example <../../dem/rotating-drum/small-scale-rotating-drum.html>`_ using `lethe_pyvista_tools <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_, a Python module is based on `PyVista <https://docs.pyvista.org/>`_, built to facilitate the reading of Lethe results using `Python <https://www.python.org/>`_. 
+
+.. important::
+  This example uses the DEM files of the `Small scale rotating drum example <../../dem/rotating-drum/small-scale-rotating-drum.html>`_.
+
+.. warning::
+  For `lethe_pyvista_tools <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ to work, along with `Python 3 <https://www.python.org/downloads/>`_, the following libraries are needed: `os <https://docs.python.org/3/library/os.html>`_, `NumPy <https://numpy.org/>`_, `PyVista <https://docs.pyvista.org/>`_, and `tqdm <https://tqdm.github.io/>`_. If any of the modules are missing, use `pip <https://pypi.org/project/pip/>`_ to install it running ``pip3 install $NAME_OF_THE_MODULE`` on the terminal.
+
+Features
+----------------------------------
+- DEM simulation
+- Post-processing using `Python <https://www.python.org/>`_, `PyVista <https://docs.pyvista.org/>`_, `lethe_pyvista_tools <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_, and `ParaView <https://www.paraview.org/>`_.
+
+
+Files used in this example
+----------------------------
+
+- Parameters file for particle insertion: ``/examples/postprocessing/small-scale-rotating-drum-postprocessing/packing-rotating-drum.prm``
+- Parameters file for drum rotation: ``/examples/postprocessing/small-scale-rotating-drum-postprocessing/small-rotating-drum-dem.prm``
+- Python module for Lethe data post-processing: ``/contrib/postprocessing/lethe_pyvista_tools.py``
+- Python script using module for rotating drum post-processing: ``/examples/postprocessing/small-scale-rotating-drum-postprocessing/example_small_rotating_drum.py``
```

Comment:
Why ``/examples/postprocessing/small-scale-rotating-drum-postprocessing/example_small_rotating_drum.py``?
Is not the example in the examples/dem repository?